File: tarea1.py
# primero ejercicio
import math
class Punto:
    def __init__(self,x,y):
        self.x=x
        self.y=y
    def coord_cartesianas(self):
        return self.x, self.y
    def coord_polares(self):
        radio=math.sqrt(self.x*self.x+self.y*self.y)
        angulo=math.atan(self.x/self.y)
        return radio, angulo
    def __str__(self):
        return str(self.x)+" " +str(self.y)
punto1=Punto(2,3)
x,y=punto1.coord_cartesianas()
print(x,y)
r,a=punto1.coord_polares()
print(r,a)
print(punto1)

File: tarea1.java
package tarea2_121;

import java.lang.Math;

class Punto {
    int x, y;

    public Punto(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int[] coordCartesianas() {
        return new int[]{x, y};
    }

    public double[] coordPolares() {
        double radio = Math.sqrt(x * x + y * y);
        double angulo = Math.atan(x / (double) y);
        return new double[]{radio, angulo};
    }

    @Override
    public String toString() {
        return x + " " + y;
    }
}

public class tarea1 {
    public static void main(String[] args) {
        Punto punto1 = new Punto(2, 3);
        int[] cartesianas = punto1.coordCartesianas();
        System.out.println(cartesianas[0] + " " + cartesianas[1]);

        double[] polares = punto1.coordPolares();
        System.out.println(polares[0] + " " + polares[1]);

        System.out.println(punto1);
    }
}
